scipy/stats/_continuous_distns.py

```diff
@@ -10508,38 +10508,44 @@ def _get_support(self, b, c):
         return self.a, c
 
     def _pdf(self, x, b, c):
-        b = np.asarray(b, dtype=x.dtype)  # avoid int to negative int power
+        b = np.asarray(b, dtype=np.asarray(x).dtype)  # avoid int to negative int power
```

Comment:
This should fix most of the other CI failures. The problem is that some functions using the private methods are no careful about passing them arrays.